### PR TITLE
Don't show submenu dropdown now that side nav handles it

### PR DESCRIPTION
**Pull request by TW80000**

### CODE REVIEW

**Review by jlhughes — APPROVED** [review]
great, was thinking the same.

**Comment by zahnno** [comment]
I was investigating this as well. https://github.com/nanoPayinc/NANOPAY/pull/8860 consider this change considering it changes the behaviour for our platform. @jlhughes @kgrgreer @TW80000

File: src/foam/nanos/menu/SubMenu.js [hunk 1]
```diff
@@ -17,8 +17,7 @@ foam.CLASS({
     },
 
     function launch(X, menu, parent) {
-      var view = this.createView(X, menu, parent);
-      view.open();
+      // noop
```
Comment (kgrgreer) [inline src/foam/nanos/menu/SubMenu.js:20]: Will this break the old style menus? Do we plan on supporting both styles?

Comment (TW80000) [reply src/foam/nanos/menu/SubMenu.js:20]: Yes, and I don't know. Probably safer to go with @zahnno's fix.

**Comment by TW80000** [comment]
Closing in favour of https://github.com/nanoPayinc/NANOPAY/pull/8860.

**Pull request closed without merging**